packages/patternfly-3/react-console/src/DesktopViewer/MoreInformationDefaultContent.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import MoreInformationInstallVariant from './MoreInformationInstallVariant';
+
+const MoreInformationDefaultContent = () => (
+  <div>
+    <p>
+      Clicking &quot;Launch Remote Viewer&quot; will download a .vv file and launch <i>Remote Viewer</i>
```

Comment:
we should provide this text as a prop or a default so that it can be internationalized.

Reply:
I'm not sure about this particular one. The `MoreInformationDefaultContent` is intended as a default fallback, the user can provide it's own content and so handle i18n this way.

Considering recent inner composition of the component, proper i18n will be complicated without access to localization tool/library - texts are concatenated there.

If content needs to be localized, can we depend on providing the whole text as a child component?

Reply:
we don't typically approach it this way in pf-react, i.e. all stateless components should provide their text as props (it is a good distinction though you make about overriding w/ children or using another component). I believe it is a good standard though to keep them as props at least in this library - simply b/c it makes the stateless components more reusable in nature (otherwise, we are implying specific use case). So for this reason, I prefer to provide those as props. Hope it helps!